5-Heroku/server/controllers/todoController.js

```diff
@@ -0,0 +1,49 @@
+const {
+    modelGetTodoById,
+    modelGetAllTodos,
+    modelPostTodo,
+    modelDeleteTodo,
+    modelUpdateTodo} = require('../models/todoModel');
+const {v4: uuidv4} =  require('uuid');
+
+const getTodoById = (req, res) => {
+    modelGetTodoById(req, (todo) => {
+        res.status(200).json(todo)
+    });
+}
+const postTodo = (req, res) => {
+    modelPostTodo(req, () => {
+        res.sendStatus(201);
+    })
+}
+
+const getAllTodos = (req, res) => {
+    const cookieId = req.cookies['user_id'];
+    if (!cookieId) {
```

Comment:
u have an assumption here that getItems is the first thing that happens and so if the user doesn't have a cookie it will create it here.

this is what known to be implicit we always want to be explicit.

what I want you to do is to write a middleware that for each request to the server it will run this logic. it will later help us in authorization as well